### DISCUSSION TITLE
How to make docker-tempo-debug work?

### DISCUSSION THREAD

**Discussion by mghildiy** [grafana/tempo#2361] [General]
I am following [this](https://github.com/grafana/tempo/tree/main/example/docker-compose/debug).

I am on Windows11, make 4.3 installed.

I try to run following command at root of the project:

`make docker-tempo-debug`

But I see:

```
FIND: Parameter format not correct
FIND: Parameter format not correct
File not found - "*.yaml"
no Go files in D:\work\learning\opensource\tempo
process_begin: CreateProcess(NULL, uname -s, ...) failed.
Makefile:51: pipe: Bad file descriptor
COMPONENT=tempo C:/ProgramData/chocolatey/lib/make/tools/install/bin/make.exe docker-component-debug
'COMPONENT' is not recognized as an internal or external command,
operable program or batch file.
make: *** [Makefile:174: docker-tempo-debug] Error 1
```

Seems I am running the command in wrong directory.

What is correct directory to run this command? I tried running it in example\docker-compose\debug, which shows:

`make: *** No rule to make target 'docker-tempo-debug'.  Stop.`

**Comment by mapno** [comment]
cc/ @stoewer

**Comment by stoewer** [comment]
The correct command to build the Tempo debug docker image is:

```
make docker-tempo-debug
```

However, the Tempo Makefile does not work on Windows at the moment. 

It uses commands that are not available on Windows or expect different parameters such as `find` or `uname`. The Makefile also uses a syntax to set environment variables for single commands that do not work on Windows like `CGO_ENABLED=0 go build …`.
